packages/react/src/SpaceFinder.tsx

```diff
@@ -24,18 +24,18 @@ export function SpaceFinder ({ spaces, selected, setSelected, className }: Space
       )
 
   return (
-    <div className={className}>
+    <div className={`${className} w3ui-space-finder-wrapper`}>
```

Comment:
maybe use the no suffix class name for the root element like
```suggestion
    <div className={`${className} w3ui-space-finder`}>
```

Reply:
which is not normally a relevant metric, but if we want devs to feel nice when they customise our thingers it becomes more important!

Reply:
agreed! I think there was a case in another component where I felt like `-wrapper` made sense, but this ain't it